.travis.yaml

```diff
@@ -0,0 +1 @@
+script: exit 0

```

Comment:
This isn't going to work.  We need to run tests for this stuff.  Check out https://github.com/okta/samples-php/blob/develop/.travis.yml for what is being done in other languages.

Reply:
We're doing the same for other .NET samples. We do have e2e tests but I'm not sure when Vijet runs those tests.